src/Microsoft.Extensions.Logging.Configuration/LoggingConfiguration.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Extensions.Configuration;
+
+namespace Microsoft.Extensions.Logging.Configuration
+{
+    internal class LoggingConfiguration
```

Comment:
This is so we don't assume that we can just take `IE<IConfiguration>` right?

Reply:
Yep, I don't want random IConfiguration's in DI, especially if they were passed into logging.